test/src/test_utils.c

```diff
@@ -335,3 +336,44 @@ ts_debug_allocated_bytes(PG_FUNCTION_ARGS)
 
 	PG_RETURN_UINT64(MemoryContextMemAllocated(context, /* recurse = */ true));
 }
+
+TS_TEST_FN(ts_test_errdata_to_jsonb)
+{
+	ErrorData *edata = (ErrorData *) palloc0(sizeof(ErrorData));
```

Comment:
Since you're filling in all fields, you do not strictly need to zero the memory. Feel free to ignore.

```suggestion
	ErrorData *edata = (ErrorData *) palloc(sizeof(ErrorData));
```

Reply:
Done!